public/js/controllers/homepage_controller.js

```diff
@@ -173,7 +173,7 @@ export default class extends Controller {
     this.windowIndexTarget.textContent = ex.window_idx
     this.posBarTarget.style.width = `${(ex.window_idx / ex.params.window_size) * 100}%`
     this.poolSizeTarget.innerHTML = humanize.decimalParts(ex.pool_info.size, true, 0)
-    this.targetPctTarget.textContent = parseFloat(ex.pool_info.percent_target).toFixed(2)
+    this.targetPctTarget.textContent = parseFloat(ex.pool_info.percent_target - 100).toFixed(2)
```

Comment:
@njirap You might need to add the `%` at the end of this line.

Reply:
This is probably what buck meant.

Reply:
She fixed it in the template, which is better